apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/OpenApiDocs.java

```diff
@@ -0,0 +1,183 @@
+package gov.cms.bfd.server.war;
+
+import static gov.cms.bfd.DatabaseTestUtils.HSQL_SERVER_PASSWORD;
+import static gov.cms.bfd.DatabaseTestUtils.HSQL_SERVER_USERNAME;
+import static io.restassured.RestAssured.certificate;
+import static io.restassured.RestAssured.given;
+
+import gov.cms.bfd.DatabaseTestUtils;
+import io.restassured.authentication.AuthenticationScheme;
+import io.restassured.authentication.CertificateAuthSettings;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.specification.RequestSpecification;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Set;
+import org.hsqldb.jdbc.JDBCDataSource;
+
+/** Program to gather the OpenAPI yaml content for V1 and V2 and store. */
+public class OpenApiDocs {
+
+  /** The HSQL database url. */
+  private static final String INMEM_HSQL_DATABASE_URL = "jdbc:bfd-test:hsqldb:mem";
+
+  /** The version from the project pom. */
+  private final String projectVersion;
+
+  /** The destination directory for the output of the OpenAPI yaml file documents. */
+  private final String destinationDir;
+
+  /** The base server url. */
+  private static String baseServerUrl;
+
+  /** The request specification to use with the api-docs requests. */
+  private static RequestSpecification requestAuth;
```

Comment:
Why are these static?

Reply:
originally the setup was static but not any longer, so I'll change it